chart/templates/deployment.yaml

```
@@ -50,6 +50,9 @@ spec:
             - "--allocate-node-cidrs=true"
             - "--cluster-cidr={{ $.Values.networking.clusterCIDR }}"
             {{- end }}
+            {{- if (gt $.Values.replicaCount 1) }}
```

Freestyling helm templates without testing it never works, this one is on me, sorry.

```suggestion
            {{- if (gt (int $.Values.replicaCount) 1) }}
```

Current version fails e2e tests with:

```
std out err: %!(EXTRA *errors.errorString=Error: template: hcloud-cloud-controller-manager/templates/deployment.yaml:53:20: executing "hcloud-cloud-controller-manager/templates/deployment.yaml" at <gt $.Values.replicaCount 1>: error calling gt: incompatible types for comparison
```

---

implemented, sorry i didn't saw the error